Update install.qmd (#1642)
Deltares/Ribasim: docs/install.qmd

Ribasim is typically used as a command-line interface (CLI). It is distributed as a `.zip`
archive, that must be downloaded and unpacked. It can be placed anywhere, however it is
important that the contents of the zip file are kept together in a directory. The Ribasim

> executable is in the main folder. 

Reviewer: ```suggestion
executable is in the main folder.
```

> executable is in the main folder. 

Reviewer 2: ```suggestion
executable is in the root folder. 
```

> To check whether the installation was performed successfully, open 'cmd' go to the path where the executable is for example 'C:\Ribasim9\ribasim_windows' and and type `ribasim` with no arguments in the command line.

Reviewer: ```suggestion
To check whether the installation was performed successfully, open a terminal and go to the path where the executable is for example 'C:\Ribasim\ribasim_windows' and and type `ribasim` with no arguments in the command line.
```
